packages/clerk-js/src/ui/common/Billing/ManageBillingPlanScreen.tsx

```diff
@@ -104,119 +103,119 @@ export const OrganizationPlanCard = (params: OrganizationPlanCardProps) => {
       elementDescriptor={descriptors.billingPlanCard}
       elementId={descriptors.billingPlanCard.setId(params.planKey)}
       sx={t => ({
-        backgroundColor: params.isCurrentPlan ? t.colors.$neutralAlpha25 : 'white',
-        width: '37.25rem',
-        borderRadius: '0.5rem',
-        boxShadow: params.isCurrentPlan
-          ? `0px 0px 0px 1px ${t.colors.$neutralAlpha100}`
-          : '0px 0px 0px 1px rgba(25, 28, 33, 0.06), 0px 1px 2px 0px rgba(25, 28, 33, 0.12), 0px 0px 2px 0px rgba(0, 0, 0, 0.08);',
+        backgroundColor: params.isCurrentPlan ? t.colors.$neutralAlpha25 : t.colors.$colorBackground,
+        borderRadius: t.radii.$lg,
```

Comment:
Is having this custom boxShadow an issue ? Are we bringing it back ? Do we need a shadow that already exists in the theme ?